packages/css-property-parser/lib/Standard.re

```diff
@@ -253,16 +257,39 @@ let interpolation = {
   Match.return(path);
 };
 
-/* TODO: LEFT_BRACKET and RIGHT_BRACKET are wrongly lexed as DELIM('[') and DELIM(']'), we use a handmade parser to by-pass it, and use LEFT_BRACKET/RIGHT_BRACKET correctly. The original spec is "'[' [ <custom-ident> ]* ']'" */
-let line_names = {
-  open Rule;
-  open Rule.Let;
-
-  let.bind_match left = Pattern.expect(LEFT_BRACKET);
-  let.bind_match path = Modifier.one_or_more(custom_ident);
-  let.bind_match right = Pattern.expect(RIGHT_BRACKET);
+let media_type =
+  token(
+    fun
+    | IDENT(value) => {
+        switch (value) {
+        | "only"
+        | "not"
+        | "and"
+        | "or"
+        | "layer" =>
+          Error([
+            "'only', 'not', 'and', 'or', and 'layer' are invalid media types",
```

Comment:
yo, give me a proper error message. 

can you do a `as used` and Printf.sprintf "%s is not a valid media type" used?

Reply:
hahaha, will fix this